go/Makefile.debian11

```diff
@@ -0,0 +1,14 @@
+IMAGES         := base main darwin arm armhf armel ppc s390x darwin-arm64 npcap
```

Comment:
Some packages are not available for `mips` in `bullseye`, so I removed them from the list of supported platforms for Debian 11